### ISSUE TITLE
wrapper doesn't wrap function declaration

### ISSUE THREAD

**Issue by dedesite** [vlang/c2v#60]
When trying to use `wrapper` command on header files, I realize that `wrapper` doesn't wrap function declaration but needs a function definition.

It would be very convenient if it were able to create wrapper only based on function declaration.

Example `test.h` : 
```c
void test();
```

`c2v wrapper test.h` Sould lead to :
```v
[translated]
module .

fn C.test() 

pub fn test()  {
	C.test()
}
``` 
But it lead to nothing for now.

```v
[translated]
module .
```

**Comment by dedesite** [comment]
Should be fixed by #78

**Comment by ArtemkaKun** [comment]
We need to add unit tests for that case
